store/src/main/java/org/apache/rocketmq/store/CommitLog.java

```diff
@@ -1045,7 +1045,7 @@ public void run() {
 
             while (!this.isStopped()) {
                 try {
-                    this.waitForRunning(0);
+                    this.waitForRunning(1000);
```

Comment:
`GroupCommitService` will be used when broker is in `SYNC_FLUSH`, so the flushCommitLogService need always `doCommit` without a break.

And this won't cause 100% CPU usage unless your server only has one cpu core.

BTW, how about use `ASYNC_FLUSH` if you mind.